spec/Store.spec.js

```diff
@@ -70,7 +70,6 @@ const mockTodoData = {
     id: '1',
     type: 'todos',
     attributes: {
-      id: '1',
```

Comment:
id as an attribute is counter the jsonapi spec.

https://jsonapi.org/format/#document-resource-object-fields